src/App.js

```diff
@@ -1,28 +1,24 @@
 import React from "react";
 import logo from "./logo.svg";
 import "./App.css";
-import Number from "./Number.js";
+import Button from "./Button.js";
```

Comment:
As discussed, please use `'` instead of `"` and drop the extension of the file in imports.